### PR TITLE
Modify sonata-project/exporter requirement to ^1.7.1

### LINKED ISSUE #4841: We should depend upon sonata-exporter ^1.7.1 because 1.7 is broken
## Subject
We should depend upon sonata-exporter ^1.7.1. Prior to that release, the bundle compiler pass add a call to [addWriter](https://github.com/sonata-project/exporter/blob/1.7.0/src/Bridge/Symfony/DependencyInjection/Compiler/ExporterCompilerPass.php#L36) which is [still private](https://github.com/sonata-project/exporter/blob/1.7.0/src/Exporter.php#L89).

### PR DESCRIPTION
**Pull request by gremo**
Closes #4841 

```markdown
### Fixed
-  exporter-related error during cache:clear command.
```

### CODE REVIEW

**Comment by greg0ire** [comment]
Please add a changelog note

**Comment by gremo** [comment]
It's ok now?

**Comment by greg0ire** [comment]
Please don't change the file itself, otherwise it will conflict with all others PRs that do the same. I'll copy the changelog to the body of your PR.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.28.1](https://github.com/sonata-project/SonataAdminBundle/compare/3.28.0...3.28.1)
+### Changed
+-  sonata-project/exporter requirement bumped to ^1.7.1
+
```
Comment (greg0ire) [inline CHANGELOG.md]: Please remove this

**Comment by greg0ire** [comment]
Please write a changelog directed at end users. Keep in mind that they probably do not care what file or class you changed, but what features you provided / what bugs you fixed. Try to answer the question "What benefit do I get from upgrading?"
More on http://keepachangelog.com

File: CHANGELOG.md (hunk 1 above)
Comment (greg0ire) [inline CHANGELOG.md]: Please remove this, I moved it to the body of your PR.

**Comment by gremo** [comment]
@greg0ire can you manage/fix it? I removed the changelog

**Comment by greg0ire** [comment]
Thanks @gremo , I fixed the changelog

**Pull request merged**